spec/models/rss_feed/rss_feed_spec.rb

```diff
@@ -80,6 +80,7 @@
 
     it "when the yaml file is updated" do
       RssFeed.seed
+      original_time = RssFeed.where(:name => @name).first.yml_file_mtime
```

Comment:
`RssFeed.where(:name => @name).first` can be simplified to `RssFeed.find_by(:name => @name)`